gulpfile.js

```diff
@@ -9,11 +9,6 @@ var buildPath = './demo/build/';
 
 gulp.task('auto-reload', loadLmnTask('auto-reload'));
 
-gulp.task('html', loadLmnTask('html', {
-  langBase: 'component.flick',
-  imagePath: '../../src/imgs/'
-}));
-
```

Comment:
How do I stop it butchering the urls in the partial? It is mkaing the urls look like this:
`../..imgs/letters/http://img.x/ala...` etc